src/__tests__/extractMeta.js

```diff
@@ -434,4 +434,109 @@ describe('Metadata extractor', () => {
             type: 'readOnly'
         });
     });
+
+    it('Extracts metadata using provided ui label', () => {
+        const props = {
+            devName: {
+                type: ElementPropTypes.string,
+                uiLabel: 'Ui label'
+            },
+            devNameTwo: {
+                type: ElementPropTypes.string,
+                uiLabel: 'Ui label two'
+            }
+        };
+
+        const extracted = extractMetadata(props);
+
+        expect(extracted).toEqual({
+            'Ui label': {
+                propName: 'devName',
+                type: 'string'
+            },
+            'Ui label two': {
+                propName: 'devNameTwo',
+                type: 'string'
+            }
+        });
+    });
+
+    it('Extracts metadata using provided ui label - mixed with non label', () => {
+        const props = {
+            devName: {
+                type: ElementPropTypes.string,
+                uiLabel: 'Ui label'
+            },
+            devNameTwo: {
+                type: ElementPropTypes.string,
+                uiLabel: 'Ui label two'
+            },
+            devNameThree: ElementPropTypes.number,
+            anIframe: {
+                type: ElementPropTypes.embeddable({
+                    embedType: {
+                        type: ElementPropTypes.string,
+                        uiLabel: 'My embed label'
+                    },
+                    url: ElementPropTypes.string,
+                    height: ElementPropTypes.number
+                }),
+                uiLabel: 'External Iframe'
+            },
+            colors: {
+                type: ElementPropTypes.shape({
```

Comment:
I'm happy to see a `shape` ElementPropType in this test, but I'd also like to see an `arrayOf` since `shape` must be nested within it.

Reply:
Sure, I can add one.

Reply:
@chris-decker-volusion Updated to `label` and added your suggested test: 3fdb6e94acc40934002cf958748052b4a52d470e